pages/contacts/_edit.vue

```diff
@@ -0,0 +1,443 @@
+                    v-model="contact.notes"
+                  />
+                </BFormGroup>
+              </div>
+              <div class="form-group">
+                <BFormGroup
+                  label="To copy full contact details into a document, click the button below, and then paste copied text into your document(CTRL-V)."
+                >
+                  <button type="button" class="btn btn-secondary" @click="copyToClipboard">
+                    Copy contact details to clipboard
+                  </button>
+                </BFormGroup>
+              </div>
+            </div>
+          </div>
+          <div class="card">
+            <div class="card-header">
+              <strong>Record Status</strong>
+            </div>
+
+            <div class="card-body">
+              <div class="form-group">
+                Created On: {{ contact.createdOn | datetime }} by {{ contact.createdBy }}
+              </div>
+              <div class="form-group">
+                Updated On: {{ contact.updatedOn | datetime }} by {{ contact.updatedBy }}
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </BForm>
+  </div>
+</template>
+
+<script>
+import {
+  BFormInput, BFormGroup, BFormSelect, BFormCheckbox, BFormTextarea,
+  BFormInvalidFeedback, BForm, BFormDatepicker, BInputGroup, BInputGroupAppend,
+} from 'bootstrap-vue';
+import Copy from 'copy-to-clipboard';
+import CountrySelector from '~/components/controls/selectors/CountrySelector';
+import RepeatedFormInput from '~/components/controls/RepeatedFormInput';
+
+const TITLE_OPTIONS = [
+  { value: null, text: '' },
+  { value: 'Mr.', text: 'Mr.' },
+  { value: 'Ms.', text: 'Ms.' },
+  { value: 'H.E. Mr.', text: 'H.E. Mr.' },
+  { value: 'H.E. Ms.', text: 'H.E. Ms.' },
+];
+
+export default {
+  name      : 'EditContact',
+  components: {
+    BFormInput,
+    BFormGroup,
+    BFormSelect,
+    BFormCheckbox,
+    BFormTextarea,
+    CountrySelector,
+    BFormInvalidFeedback,
+    BForm,
+    RepeatedFormInput,
+    BFormDatepicker,
+    BInputGroup,
+    BInputGroupAppend,
+  },
+  data() {
+    return {
+      contactId    : this.$route.params.edit || '',
+      contact      : {},
+      inputBoxId   : 0,
+      titleOptions : [ ...TITLE_OPTIONS ],
+      emailRequired: false,
+    };
+  },
+  computed: {
+    firstNameRequired() { return (this.contact.firstName || '').trim() !== ''; },
+    lastNameRequired() { return (this.contact.lastName || '').trim() !== ''; },
+    countryRequired() { return !this.contact.country && !this.contact.isUseOrganizationAddress; },
+  },
+  mounted,
+  methods: {
+    copyToClipboard,
+    contactUpdate,
+    goBack,
+  },
+};
+
+async function mounted() {
+  const result = await this.$kronosApi.getContact(this.contactId);
+  this.contact = result;
+  if (result.emails && result.emails.length > 0) {
+    this.emailRequired = true;
+  }
+}
+
+async function contactUpdate(event) {
```

Comment:
We have standard naming convention. please rename to `save`
```suggestion
async function save(event) {
```

Reply:
DONE ! renamed name as suggested